src/components/services/EdgeContextCallbackManager.tsx

```diff
@@ -39,9 +39,11 @@ class EdgeContextCallbackManagerComponent extends React.Component<Props> {
 
         if (!errorShown) {
           errorShown = true
-          this.showError(error).then(() => {
-            errorShown = false
-          })
+          this.showError(error)
+            .then(() => {
+              errorShown = false
+            })
+            .catch(err => showError(err))
```

Comment:
The `this.showError` method should return `void` instead of `Promise<void>`, and should do it's own catching. That way, it works more like our other logging code.

Reply:
But then we'd remove the logic to only show one error every 5 seconds (duration of the dropdown).

Reply:
In that case, maybe the singleton logic should live inside of `this.showError` too.